tests/test_manual_connect.c

```diff
@@ -117,14 +155,15 @@ void test_connect_with_duo_passcodeInPassword(void** unused)
     snowflake_term(sf);
 }
 
-int main(void)
+int main(void) 
 {
     initialize_test(SF_BOOLEAN_FALSE);
     const struct CMUnitTest tests[] = {
```

Comment:
Can we use env var (ex. "SNOWFLAKE_MANUAL_TEST_TYPE") to select which test we want to run?

Reply:
Done.